CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # 8.2.2 (not released)
 
+### Added
+- Cardano: `getNativeScriptHash` call
+- Cardano: support for 1854 and 1855 key paths
+- Cardano: support for script addresses
+- Cardano: support for token minting and burning
+- Cardano: support for multi-sig transactions using native scripts
```

Comment:
```suggestion
- Cardano: support for multi-sig transactions using native scripts in stake credentials
```

Reply:
Isn't the current formulation correct? I mean, we do add support for multi-sig transactions using native scripts in payment credentials as well, don't we?

Reply:
True, I don't understand why I suggested a change.